web/src/engine/websites/ManhuaPlus.ts

```diff
@@ -1,20 +1,19 @@
 import { Tags } from '../Tags';
 import icon from './ManhuaPlus.webp';
 import { DecoratableMangaScraper } from '../providers/MangaPlugin';
-import * as Madara from './decorators/WordPressMadara';
 import * as Common from './decorators/Common';
+import * as Liliana from './decorators/Liliana';
 
-@Madara.MangaCSS(/^{origin}\/manga\/[^/]+\/$/)
-@Madara.MangasMultiPageCSS()
-@Madara.ChaptersSinglePageAJAXv2()
-@Madara.PagesSinglePageCSS('div.reading-content img')
-@Common.ImageAjax()
+@Common.MangaCSS(/^{origin}\/manga\/[^/]+$/, 'article header h1')
+@Common.MangasMultiPageCSS(Liliana.mangaPath, 'section div.text-center a.clamp', 1, 1, 0, Common.AnchorInfoExtractor(true))
+@Common.ChaptersSinglePageCSS(Liliana.queryChapters)
+@Common.PagesSinglePageJS(Liliana.queryPagesScript, 500)
+@Common.ImageAjax(true, true)
 export default class extends DecoratableMangaScraper {
 
     public constructor() {
-        super('manhuaplus', 'ManhuaPlus', 'https://manhuaplus.com', Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Language.English);
+        super('manhuaplus', 'ManhuaPlus', 'https://manhuaplus.org', Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Language.English, Tags.Source.Aggregator);
```

Comment:
The https://manhuaplus.com Madara website is still available. The https://manhuaplus.org website seems to be a different one. Shouldn't this be a new website instead of replacing the existing one?

Reply:
Same owner.  For a while we can see that on the .com website.
![image](https://github.com/user-attachments/assets/83ddce85-c1a7-4fcc-ba38-0adaad600aba)